.circleci/config.yml

```diff
@@ -34,7 +34,7 @@ jobs:
             . /opt/conda/etc/profile.d/conda.sh
             conda activate esmvaltool
             pip install .[test]
-            pytest -n 2 -m "not installation and not sequential"
+            pytest -n 2 -m "not sequential"
```

Comment:
Can you briefly explain why these changes in this file are relevant for this PR?

Reply:
thanks for looking into this, man! Yes, good point, I should add this to the PR description: the `not installation` message tells pytest not to run the bits that need NCL, R, Julia etc if the executables are not on the system, so that those tests don't fail; that reduces task testing by quite a bit, and `_task.py` test coverage is pretty bad that way, so I went about and made it do those checks anyway, without the tests failing if the executables are not found

Reply:
also see my comment here (I actually did bother commenting on that, but I didn't include it in the PR description) https://github.com/ESMValGroup/ESMValCore/pull/1404#issuecomment-988168030